content/blog/2021-11-05.md

```diff
@@ -0,0 +1,206 @@
+
+The jmp.chat service blog [announced](https://blog.jmp.chat/b/october-newsletter-2021) some changes to the SIP inbound calling feature, so users are advised  to configure it via the new interactive bot. On the iOS side Snikket is now the recommended app as some helpful functions (like calling tones) got sponsored.
+
+Aria Network wrote a follow-up blog post on [UX improvements such as avatars of the Bifrost bridge to Matrix](https://aria-net.org/SitePages/Bifrost-Improvements.aspx). The previous article was ["Conversion of communities to spaces, new Bifrost features"](https://aria-net.org/SitePages/Regarding-spaces-and-Bifrost.aspx).
+
+## Software news
+
+### Clients and applications
+
+Following last month's Conversations and Quicksy releases of 2.10.0, a [2.10.1 point release](https://github.com/iNPUTmice/Conversations/releases/tag/2.10.1) comes with a new internal library for video compression, which allows more devices to send smaller files. Conversations will now be included in Android backups (be it Seedvault or Google's) but only if encrypted. Furthermore, there have been address book fixes so contact addresses are picked up easier. Finally there have been fixes for notifications, which allow to  start downloads properly and send MUC PMs to the right contact.
+
+[Gajim 1.3.3 has been released](https://gajim.org/post/2021-10-10-gajim-1.3.3-released/). This release features improved Ad-Hoc Commands and brings back spell checking. Gajim 1.3.3 includes many bug fixes and improvements. Also this month: two years of Gajim Development News! In October, Gajim’s event system received significant changes. These changes lead to improvements in notifications mainly, but also made it possible to display Jingle File Transfers directly in the chat view. Another nice addition: search filters which help narrow down your search.
+
+[Version 2.10.2 of the Snikket Android app has been released](https://fosstodon.org/@snikket_im/107161786225773950)! The main highlight of this release is the new dialpad which allows you to navigate automated call systems when using a Jingle to PSTN bridge.
+
+New beta versions of [Beagle IM and Siskin IM have been released](https://mastodon.technology/@tigase/107161741669699519) for MacOS and iOS with support for location sharing.
+
+## Servers
+
+The Openfire plugin for Push Notification got a small bug fix, so server admins are advised to update to [the new 0.9.0 version](https://discourse.igniterealtime.org/t/push-notification-openfire-plugin-0-9-0-released/90934).
+
+## Libraries
+
+Development on mellium.im/xmpp (an XMPP library in Go) has been slower than usual this month. We've primarily been focused on rolling out basic pubsub, PEP, and PEP Native Bookmarks support and the basics are already merged into the main branch. For more information, see the [latest Dev Communiqué](https://opencollective.com/mellium/updates/dev-communique-for-october-2021).
```

Comment:
Maybe link `mellium.im/xmpp` to https://mellium.im/xmpp or https://pkg.go.dev/mellium.im/xmpp?

Reply:
1c7bf93